public/css/list/list-item.less

```diff
@@ -23,6 +23,8 @@
   }
 
   .caption {
+    overflow: hidden;
+
```

Comment:
Please remove this. For one thing this is the style section of this file and this doesn't belong here. Also, you've removed the same instruction in file `item-list.less`. Why not keep it there instead?

Reply:
✅